auth-web/src/components/auth/mixins/NextPageMixin.vue

```diff
@@ -49,7 +49,9 @@ export default class NextPageMixin extends Vue {
   protected getNextPageUrl (): string {
     switch (this.currentUser?.loginSource) {
       case LoginSource.IDIR:
-        return `/${Pages.SEARCH_BUSINESS}`
+        let idirNextStep = '/'
```

Comment:
Don't this change is necessary? Maybe leftover from something?

Reply:
I will change that